supernode/examples/configs/localnet-4444.yml

```diff
@@ -13,5 +13,6 @@ node:
 p2p:
   listen_addresses: "0.0.0.0"
   port: 4454
+  data_source_name: ":memory:"
```

Comment:
```suggestion
  # default value is assigned automatically, by concatenated flag `--work-dir` and `p2p` as a subfolder
  data_dir: "path/to/p2p/data/dir"

  # if not specified, persistent database store is used
  memory_db: true

 ```

Reply:
sorry I misunderstand here for config files :) thanks for fixing them.